src/backends/custom/custom_backend.cc

```diff
@@ -459,8 +463,25 @@ CustomBackend::Context::GetNextInput(
   const std::string name(cname);
   Scheduler::Payload* payload = input_context->payload_;
 
+  // If the memory type is on GPU, implicitly copying it to CPU memory
+  // to ensure backward capability
```

Comment:
Make sure to file a DLIS about us needing to extend custom backend API to allow input/output tensors in GPU memory